### PR TITLE
hw1

### PR DESCRIPTION
**Pull request by vsevolodvitkovski**

### CODE REVIEW

File: homework1/vsevolod_vitkovski.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def interliner(x, y, z):
+
```
Comment (pbcmf) [inline homework1/vsevolod_vitkovski.py:6]: It should have been a class.

File: homework1/vsevolod_vitkovski.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import matplotlib.pyplot as plt
+
+
+def interliner(x, y, z):
+    
+    return np.interp(z, x, y)
+    
+def test(x, y, z):
+    
+    plt.plot(x, y)
```
Comment (pbcmf) [inline homework1/vsevolod_vitkovski.py:11]: The purpose of the test is not to visually show the value, but to test the edge cases, including exceptions and failing tests as well as one that perform ok.

File: homework1/vsevolod_vitkovski.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    plt.plot(z, interliner(x, y, z), marker="o", markersize=10, markeredgecolor="red")
```
Comment (pbcmf) [inline homework1/vsevolod_vitkovski.py:12]: Comments should be added the test and file

File: homework1/vsevolod_vitkovski.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
```
Comment (pbcmf) [inline homework1/vsevolod_vitkovski.py:3]: Cannot merge your pull since:
1) You have deleted the file polina_baranovva.py - which you have not. This was pushed earlied and should be considered as integral part of lib. You can change/modify files only related to your piece of works, this is how git and collaboration work.
2) Please remove binary files and I do not believe they are needed (looks like system filed)

**Review by pbcmf — CHANGES_REQUESTED** [review]
